server/routes/reviewsRoutes.js

```diff
@@ -2,29 +2,15 @@ const express = require('express');
 const router = express();
 const rev = require('./reviewsHelpers');
 
-
-router.get('/reviews', (req, res) => {
-  let productId = req.query.id;
-  rev.getReviews(productId)
-    .then((product) => {
-      res.send(product.results);
-    })
-    .catch((err) => {
-      console.log('Err: ', err);
-    });
-});
-
-router.get('/meta', (req, res) => {
-  let productId = req.query.id;
-  let sortOption = req.query.sort;
-  rev.getMetaData(productId, sortOption)
+router.put('/helpful', (req, res) => {
+  let reviewId = req.body.revId;
+  rev.addHelpfulVote(reviewId)
     .then((data) => {
-      res.send(data)
+      res.sendStatus(data)
```

Comment:
I'm just curious. I see you replace res.send with res.sendStatus. Is res.sendStatus what you want to do with a put request?

Reply:
Oh yeah! I don't actually need anything back from a put request but i didn't know what to do so I just did res.sendStatus. maybe I could do res.end?

Reply:
Sounds good, I was just curious since I've never used a put request. I'm sure you're good with just sending a status code back if no data needs to be sent back.